### DISCUSSION TITLE
Add number of links near crate name

### DISCUSSION THREAD

**Discussion by Uni-versalis** [WebCrateApp/webcrate#33] [Ideas]
Small idea: add the number of links in a crate near it's name on the left panel

**Comment by BetaHuhn** [comment]
The number is already displayed on the home page for recently used crates and I see why it might be useful to display it in the sidebar as well.

The only concern I have is that the space on the sidebar is quite limited and it may appear even more crowded with a number next to the crate name. Will see if I can design something that isn't too big/distracting.
